exercises/concept/making-the-grade/.meta/design.md

```diff
@@ -0,0 +1,38 @@
+# Design
+
+## Learning objectives
+
+- Know how to repeteadly execute code using a `for` loop.´
+- Know how to repeteadly execute code using a `for-each` loop.
+- Know how to repeteadly execute code using a `while` loop.
+- Know how to repeteadly execute code using a `do-while` loop.
+
+## Out of scope
+
+- Specific iteration over a `Map`
```

Comment:
Why is this listed here? When I think about loops I don't necessarily think of maps.

Things that are related and seem to be out of scope of this exercise:

- Breaking out of loops using the `break` keyword
- Skipping iterations using the `continue` keyword

Reply:
Instead of adding those to the out of scope I prefer mention them and maybe try to add exercise to cover those topics now that you say it, do you have in mind some other topics that could be out of scope? or I should remove it?

Reply:
6 tasks in a not very complex concept for me seems acceptable